### DISCUSSION TITLE
setting up Automatonism with PlugData

### DISCUSSION THREAD

**Discussion by zanevsky-null** [plugdata-team/plugdata#199] [Q&A]
Hello! I have PlugData running nicely inside Bitwig. I noticed in the screencap on the PlugData page that the Maestro mixer from Automatonism is running. I tried opening the Automatonism main.pd file in PlugData but had no luck. How do you get these modules to run in PlugData? Thanks for reading and for the extraordinary work you've done here.

**Comment by zanevsky-null** [comment]
> Hello! I have PlugData running nicely inside Bitwig. I noticed in the screencap on the PlugData page that the Maestro mixer from Automatonism is running. I tried opening the Automatonism main.pd file in PlugData but had no luck. How do you get these modules to run in PlugData? Thanks for reading and for the extraordinary work you've done here.

I actually can open main.pd in PlugData but can't access the modules list or call any modules.

**Comment by timothyschoen** [comment]
Hi! What I did is just put the "patch_editor_abs" folder from automatonism in PlugData's search path. 

I'm not sure if all modules work, I've tested basic-lfo, basic-osc, maestro and a few others with no problems so far.

**Reply by zanevsky-null** [reply]
awesome thank you that worked!

**Comment by alfonso73** [comment]
Given that AFAIK Automatonism is all vanilla based i guess it should fully work with PlugData
